app/experimenter/nimbus-ui/src/components/TableSummary/index.test.tsx

```diff
@@ -146,6 +146,62 @@ describe("TableSummary", () => {
       ).not.toBeInTheDocument();
     });
   });
+
+  describe("renders 'Risk mitigation checklist' row as expected", () => {
+    it("when set", () => {
+      const { experiment } = mockExperimentQuery("demo-slug", {
+        riskMitigationLink: "https://mozilla.org",
+      });
+      render(<Subject {...{ experiment }} />);
+      const link = screen
+        .getByTestId("experiment-risk-mitigation-link")
+        .querySelector("a");
+      expect(link).toHaveTextContent(experiment.riskMitigationLink);
+      expect(link).toHaveAttribute("href", experiment.riskMitigationLink);
+    });
+    it("when not set", () => {
+      const overrides = {
+        riskMitigationLink: undefined,
```

Comment:
Will this actually be `undefined` if not set, or will it be `null`?

Reply:
According to the [types](https://github.com/mozilla/experimenter/blob/jh/4063/links-table/app/experimenter/nimbus-ui/src/types/getExperiment.ts#L100) it is only `string`, which seems incorrect. But whether it's null or undefined it'll be falsy when evaluating 🤷